recipes/m4/all/conanfile.py

```diff
@@ -102,6 +102,8 @@ def package(self):
 
     def package_id(self):
         self.info.include_build_settings()
+        del self.info.settings.arch
```

Comment:
arch is relevant for an executable (e.g. arm vs intel)

Reply:
removed - with a squash